install.m

```diff
@@ -0,0 +1,15 @@
+% This is installation file for FEX
+
+% Determine where this m-file's folder is.
+[base_dir,~,~] = fileparts(mfilename('fullpath'));
+

```

Comment:
When using only the fist output, it's better form to just omit all other outputs: `base_dir = fileparts(...)`